src/contexts/alert_handler.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { createContext, useContext, useState } from "react";
+
+const AlertHandlerContext = createContext();
+
+export const AlertHandlerProvider = ({ children }) => {
+  const [alertState, setAlertState] = useState({
+    visible: false,
+    alertMessage: "",
+  });
+
+  const setAlert = (alertMessage) => {
+    setAlertState({ visible: true, alertMessage: alertMessage });
+  };
+
+  const closeAlert = () => {
+    setAlertState({ ...alertState, visible: false });
+  };
+
+  const contextValue = {
+    ...alertState,
+    setAlert: setAlert,
+    closeAlert: closeAlert,
+  };
```

Comment:
```suggestion
    setAlert,
    closeAlert,
  };
```
プロパティ名と値に指定する変数名が同じ場合は、このように省略して書くこともできます。

参考：https://jsprimer.net/basic/object/#create-object